ctapipe/instrument/camera.py

```diff
@@ -145,7 +157,7 @@ def _calc_pixel_area(pix_x, pix_y, pix_type):
         Note this will not work on cameras with varying pixel sizes.
```

Comment:
I just noticed we never check for this case where it doesn't work... could be good to raise an exception or warning. The Whipple490 camera can be used to test that it fails (it has mixed pixel radii)

Reply:
You cannot really throw an error here, because this function is only called, when the pixel area is unknown.

Maybe we should just remove it, I think it's not used anymore. Pixel area is known from the simtel files.

Reply:
I think it was just for some of the camera definitions I collected from other telescopes where the area wasn't specified, but then it was calculated and written back out to FITS files, so perhaps it's not needed anymore.